action.yml

```
@@ -13,6 +13,9 @@ inputs:
   args:
     description: 'Arguments to pass to Dagger'
     required: false
```

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddeprecationmessage

```suggestion
  args:
    description: 'Arguments to pass to Dagger'
    required: false
    deprecationMessage: 'Use cmds input instead'
```

Later on for a major release we can remove `args`.